src/ISA/ISA/ArcTypes/CompositeHeader.fs

```diff
@@ -194,6 +192,18 @@ type CompositeHeader =
         | ProtocolType -> "DPBO:1000161"
         | anyelse -> failwith $"Tried matching {anyelse} in getFeaturedColumnAccession, but is not a featured column."
 
+    /// <summary>
+    /// This function gets the associated term accession for term columns. 
+    /// </summary>
+    member this.GetColumnAccessionShort =
```

Comment:
change name to `GetAccessionShort`? It is implemented on the header without input. I think adding `Column` makes the name more vague than removing it.

Or rename to `GetHeaderAccessionShort` at least

Reply:
Agreed, that the `column` gives no useful information here